docs/gemstash-configuration.5.md

```diff
@@ -104,6 +104,23 @@ Valid values
 
 A valid database URL for the [Sequel gem](http://sequel.jeremyevans.net/)
 
+DB Connection Options
+=====================
+
+`:db_connection_options`
+
+Specifies additional `Sequel.connect` options to use.
+
+Default value
+-------------
+
+None
+
+Valid values
+------------
+
+A valid connection options Hash for the [Sequel.connect](http://sequel.jeremyevans.net/rdoc/files/doc/opening_databases_rdoc.html#label-General+connection+options) method.
```

Comment:
Could you please move the changes to files in the `docs` directory into the corresponding files in the `man` directory? Those end up generating these :smile: